<ui version="4.0" >
 <class>SysInfoExWindowClass</class>
 <widget class="QMainWindow" name="SysInfoExWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SysInfoEx</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="sizeConstraint" >
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="page" >
       <layout class="QVBoxLayout" name="verticalLayout_4" >
        <item>
         <widget class="QLabel" name="modelLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="manufacturerLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="versionLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="languageLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imeiLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imsiLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>276</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2" >
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <widget class="QLabel" name="signalStrengthLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="batteryLevelLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="diskSpaceLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="criticalDiskSpaceLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>316</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3" >
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QListWidget" name="listWidget" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionSystemInfo" >
   <property name="text" >
    <string>System Info</string>
   </property>
  </action>
  <action name="actionSystemInfo2" >
   <property name="text" >
    <string>System Info 2</string>
   </property>
  </action>
  <action name="actionFeatureDiscover" >
   <property name="text" >
    <string>Feature Discover</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
